diff --git a/LDPIS/LDPIS-Frontend/src/views/Mill/LD15S001.jsx b/LDPIS/LDPIS-Frontend/src/views/Mill/LD15S001.jsx
index 70d2717..8e02a0a 100644
--- a/LDPIS/LDPIS-Frontend/src/views/Mill/LD15S001.jsx
+++ b/LDPIS/LDPIS-Frontend/src/views/Mill/LD15S001.jsx
@@ -1467,13 +1467,16 @@ const showErrorAlert = (errorMessage) => {
       headerFilterPlaceholder: "search...",
       headerFilter: "input",
     },
-    // {
-    //   title: "Asl No",
-    //   field: "ASL_NO",
-    //   headerFilterPlaceholder: "search...",
-    //   headerFilter: "input",
-    //   editor: "input",
-    // },
+    {
+      title: "ASL No",
+      field: "TBP_ASL_NO_80",
+      headerFilter: "input",
+    },
+    {
+      title: "Field No",
+      field: "TBP_FLD_NO_130",
+      headerFilter: "input",
+    },
     { title: "Tag", 
       field: "TBP_SAMPLE_10",
        headerFilter: "input", 
